src/metabase/integrations/slack.clj

```diff
@@ -42,11 +42,15 @@
   :visibility :internal
   :type :json)
 
+(def ^:private zoned-time-epoch (t/zoned-date-time 1970 1 1 0))
```

Comment:
```suggestion
(def ^:private zoned-1970 (t/zoned-date-time 1970 1 1 0))
```
wdyt about this name? I think it's more expressive. 
Also if you want it to be a bit poetic: `zoned-beginning-of-time`

Reply:
@qnkhuat counterpoint, the term [_Unix epoch_](https://en.wikipedia.org/wiki/Unix_time) specifically refers to 0 UTC on Jan 1 1970, so I think it's probably appropriate to use _epoch_ here as well

Reply:
ahh, agreed. TIL about that term. Thanks